src/browser/main.js

```diff
@@ -11,6 +11,9 @@ import {Provider} from 'react-redux';
 // TODO: Add app storage example.
 // import storage from 'redux-storage';
 
+// Enabling ES7 `async/await` in browser:
+if (process.env.IS_BROWSER) require('babel-core/polyfill');
```

Comment:
I see your point. Let's replace custom hash logic with Webpack based. Hmm, I think it's right. Thank you!